src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/lecture-unit-management.component.ts

```diff
@@ -139,6 +160,9 @@ export class LectureUnitManagementComponent implements OnInit, OnDestroy {
         if (lectureUnit.type === LectureUnitType.ATTACHMENT) {
             return 'artemisApp.attachmentUnit.delete.typeNameToConfirm';
         }
+        if (lectureUnit.type === LectureUnitType.VIDEO) {
+            return 'artemisApp.videoUnit.delete.typeNameToConfirm';
+        }
     }
```

Comment:
wouldn't a switch case here be more readable? Also adding a default case might be relevant?

Reply:
done in b8e9bc9f529484f9109a58503fb96a81fc84b78e